code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -109,8 +109,8 @@
 		AC.Remove(src)
 	Target = null
 	Leader = null
-	Friends.Cut()
-	speech_buffer.Cut()
+	QDEL_LIST(Friends)
+	QDEL_LIST(speech_buffer)
```

Comment:
QDEL_LIST(Friends) will delete anyone who has ever fed the slime plasma
speech_buffer is a list of lists so qdelling them will not accomplish much